packages/@vue/cli-plugin-babel/index.js

```diff
@@ -44,8 +46,20 @@ module.exports = (api, {
           .loader('thread-loader')
     }
 
-    jsRule
+    const babelLoader = jsRule
       .use('babel-loader')
         .loader('babel-loader')
+
+    // use project's babel config for dependencies
+    if (transpileDependencies.length !== 0) {
+      // `babelrc` = false
+      const config = babelHelpers.loadUsersConfig(api.resolve('.'))
+
+      // prevent from ignore
+      config.ignore = [babelHelpers.resolveNodeModulesIgnorePattern(transpileDependencies)]
```

Comment:
This is not needed because we already have the `exclude` rule above which applies `babel-loader` only to the correct files.

Reply:
yes, you are right